src/components/HoursReportFooter/HoursReportFooter.css

```diff
@@ -0,0 +1,59 @@
+.c-hours-report-footer{
```

Comment:
Make the buttons inside the container of 360px inside this container make it flex and space-evenly (between). and make this container center